pkg/security/ebpf/c/include/helpers/activity_dump.h

```diff
@@ -53,10 +53,16 @@ __attribute__((always_inline)) struct cgroup_tracing_event_t *get_cgroup_tracing
     return evt;
 }
 
+__attribute__((always_inline)) u32 is_cgroup_activity_dumps_supported(struct cgroup_context_t *cgroup) {
+    u32 cgroup_manager = cgroup->cgroup_flags & 0b111;
```

Comment:
could we have a named constant for 0x0b111 ?

Reply:
Done

Reply:
Thanks!